src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -34,6 +34,7 @@ static ManagedCompiler()
         public ManagedCompiler()
         {
             TaskResources = ErrorString.ResourceManager;
+            LogStandardErrorAsError = true;
```

Comment:
We should add a comment on why we are doing this. There isn't a great way to test this so need to have a comment justifying the change else deleting it will seem to be a fine change as it doesn't break CI